lib/galaxy/datatypes/binary.py

```diff
@@ -3759,6 +3759,35 @@ def sniff(self, filename: str) -> bool:
         return Fast5Archive.sniff(self, filename)
 
 
+class Pod5(Binary):
+    """
+    Class describing a Pod5 file
+
+    >>> from galaxy.datatypes.sniff import get_test_fname
```

Comment:
We often also have a negative test. Not sure if needed here.

Reply:
Done in 385e7ab1e1f35c596b79efa7ddfffe0d4da753e4